### PR TITLE
docs: Add validation to list of packages in readme

### LINKED ISSUE #520: Add Validation to package list in readme
The validation package is missing from the table of packages,

### PR DESCRIPTION
**Pull request by JNygaard-Skylight**
## Description

Add validation to list of packages in readme.

## Related Issues

Closes #520 

<--------------------- REMOVE THE LINES BELOW BEFORE MERGING --------------------->

## Checklist

Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.

## Checklist for Reviewers

Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ This is a **uv workspace** (Python) with a separate **npm workspace** (TypeScrip
 | [`model-tuning`](packages/model-tuning/)               | Fine-tunes SentenceTransformer models and builds HNSW indexes for OpenSearch                         |
 | [`api`](packages/api/)                                 | FastAPI service exposing `/api` endpoints; serves the built frontend in non-local environments       |
 | [`frontend`](frontend/)                                | React 19 + TypeScript + Vite demo application for interacting with the API                           |
+| [`validation`](packages/validation/)                   | Functionality to validate an eICR and to create Schematron output                                    |
```
Comment (BradySkylight) [inline README.md:71]: Is there any where that we have documented how to use this validation package or is this just only going to be part of the e2e testing?  Or maybe you leveraged the one that was part of the former eicr validation tool repo?

**Review by robertandremitchell — APPROVED** [review]
